packages/admin/cms-admin/src/dam/DataGrid/FolderDataGrid.sc.tsx

```diff
@@ -5,6 +5,11 @@ interface TableHoverHighlightProps {
     isHovered?: boolean;
 }
 
+export const FolderWrapper = styled("div")`
+    padding: ${({ theme }) => theme.spacing(4)};
+    overflow-y: scroll;
```

Comment:
Shouldn't this be `auto`? Otherwise a scroll bar will be shown all the time

Reply:
You're right thanks 👍🏻